rust/syntect-plugin/examples/make_manifest.rs

```diff
@@ -41,7 +42,7 @@ fn parse_name_and_version() -> Result<(String, String), io::Error> {
 }
 
 fn main() -> Result<(), io::Error> {
-    let syntax_set = SyntaxSet::load_defaults_newlines();
+    let syntax_set = SyntaxSet::load_from_folder("syntect-resources/Packages").unwrap();
```

Comment:
So as discussed, we also need to `add_plain_text_syntax()` here. I'm not sure if the order we do this in matters, so I would just do exactly what `gendata.rs` does?